client/css/style.css

```diff
@@ -1365,11 +1369,27 @@ background on hover (unless active) */
 }
 
 #chat .error,
-#chat .error .from,
-#chat .channel .highlight .from,
-#chat .channel .highlight .text,
-#chat .channel .highlight .user {
-	color: #f00;
+#chat .error .from {
+	color: #e74c3c;
+}
+
+#chat .channel .message.highlight {
+	background-color: var(--highlight-bg-color);
+	border-left: 5px solid var(--highlight-border-color);
+}
+
+#chat .channel .message.highlight .time {
+	padding-left: 5px;
+	width: 50px;
+	color: #696969;
+}
+
+#chat .channel .message.highlight .content {
+	border-left: 1px solid var(--highlight-bg-color);
+}
+
+#chat .channel .message.highlight .from::after {
+	background: transparent;
```

Comment:
What does this do?

Reply:
Removed. It was a relic from https://github.com/thelounge/thelounge/pull/2270/files#diff-97db1f70168fb5f12457b238ff6052b5L2230, thanks for noticing!